src/pages/linux/index.md

```diff
@@ -5,6 +5,6 @@ title: Linux
 
 Linux is the most popular operating system in the world. It powers 90% of the world's servers, and is the basis of Google's popular Android operating system.
 
-There are also versions of Linux designed to be run on personal computers, such as Ubuntu.
+There are also many versions of Linux designed to be run on personal computers.
```

Comment:
I think having Ubuntu here is fair. It's not mentioning that it's good or bad, it's saying that Ubuntu is a version of Linux designed for personal computers. It's a good example because it's still the most popular one. I think adding `many` is a good change. Adding another example is fine with me too: "such as Ubuntu and Debian."